content/blog/18-19/g2_blogpost.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "A Gateway to collaboration"
+created_at: 26-02-2019
+description: Het ProjectenProject
+author: Arne Bertrand
+---
+> Ik wil meedoen met een zeus project, hoe doe?
+
+Het is een vraag die enorm vaak voorkomt en verbazend moeilijk te 
+beantwoorden is. De keuze van het project op zich is vaak al lastig. 
+Welke technologie wil ik gebruiken? Welke projecten zijn nog relevant?
+Welk project heeft het meeste hulp nodig? Het zijn stuk voor stuk aspecten
+die meespelen en elkaar vaak tegenspreken.
+
+Eens een project gekozen, komt er vaak nog een hele boterham setup bij kijken
+die, we moeten het eerlijk toegeven, niet altijd goed gedocumenteerd is.
+Eens dan eindelijk de setup voltooid is, komt nog het moeilijkste deel:
+het selecteren van een issue die:
+
+- relevant is
+- binnen je niveau ligt
+- enigszins doenbaar is in de beperkte tijd die je kan vrijmaken
+
+
+## G2 - Gateway
+
+De oplossing? NOG een project natuurlijk! G2 wordt onze eigenste
+project-management hub. Het zal projectleiders de tools aanbieden om issues
+en branches op een logische manier te structureren. Projecten worden 
+recursief onderverdeeld in subprojecten, die elk op zich een logisch
+samenhangende collectie van issues en branches zijn. 
+
+Ja, dit kan nu ook al op zekere hoogte met zaken zoals tags op git(hub|lab). 
+De meerwaarde zit dan ook vooral in de andere richting.
+
+Nieuwe leden kunnen op een eenvoudige en gecentraliseerde manier de lopende
+projecten verkennen. Hierbij moeten ze niet steeds de repos doorspitten,
+maar kunnen ze zoeken op talen, frameworks en "soort" projecten (backend, 
+frontend, etc). 
+
+Leden met al enige pull requests achter de kiezen kunnen na een hiatus
+zich makkelijk herorienteren in hun favoriete projecten, en een 
+subproject uitkiezen dat hun aanspreekt.
+
+Het projectbestuur kan mensen met bovenstaande vraag makkelijk doorverwijzen,
+alsook een oogje houden op de status en populariteit van alle projecten.
```

Comment:
`de status en populariteit van alle projecten in het oog houden` lijkt mij een beter verwoording maar you do you